### ISSUE TITLE
Fix links to example CLI inputs

### ISSUE THREAD

**Issue by alyssadai** [neurobagel/documentation#81]
Broken links on:
- https://neurobagel.org/dictionaries/
  - also, move tip box containing examples further up
- https://neurobagel.org/graph_data/

**Comment by Remi-Gau** [comment]
Yup there are a couple of tools out there to check links in MD documents.
I can set it up if you want.

**Comment by Remi-Gau** [comment]
here is an example: https://github.com/bids-standard/bids-website/blob/gh-pages/.github/workflows/check_md_links.yml

and the associated config: 
https://github.com/bids-standard/bids-website/blob/gh-pages/md_link_check_config.json

**Comment by rmanaem** [comment]
Thanks @Remi-Gau!

**Comment by rmanaem** [comment]
@alyssadai 

>also, move tip box containing examples further up. Actually, the contents should probably be moved out of the tip box in to its own section called "Example data dictionary" or something. This section could come before or after the "Phenotypic attributes" section

I think that text is too small for section, how about we move it above phenotypic attributes and put it in an info box instead of a tip box?

**Comment by alyssadai** [comment]
Hey @rmanaem, sure, we can leave it in an info box.

The goal would be to make the examples super easy to find on the page, so -

> how about we move it above phenotypic attributes

yes, I think it should be in the Overview section, maybe under the introductory sentence?
